Kickstarter-iOS/Views/Controllers/PaymentMethodsViewControllerTests.swift

```diff
@@ -17,26 +17,26 @@ class PaymentMethodsViewControllerTests: TestCase {
   }
 
   func testView_WithCreditCards() {
-    let response = UserEnvelope<GraphUserCreditCard>(
-      me: GraphUserCreditCard.template
-    )
+    let graphUser = GraphUser.template |> \.storedCards .~ .template
+    let response = UserEnvelope<GraphUser>(me: graphUser)
 
     self.generateSnapshots(with: response)
   }
 
   func testView_NoCreditCards() {
-    let response = UserEnvelope<GraphUserCreditCard>(me: GraphUserCreditCard.emptyTemplate)
+    let graphUser = GraphUser.template |> \.storedCards .~ .emptyTemplate
+    let response = UserEnvelope<GraphUser>(me: graphUser)
 
     self.generateSnapshots(with: response)
   }
 
-  private func generateSnapshots(with response: UserEnvelope<GraphUserCreditCard>) {
+  private func generateSnapshots(with response: UserEnvelope<GraphUser>) {
     combos(Language.allLanguages, Device.allCases).forEach {
       arg in
 
       let (language, device) = arg
       withEnvironment(
-        apiService: MockService(fetchGraphCreditCardsResponse: response),
+        apiService: MockService(fetchGraphUserResponse: response),
```

Comment:
the last commit I just pushed (addressing the userEmailFields) broke a bunch of snapshots because the email address changed but that's not a big deal. again lots of tedious, time consuming small tasks but functionality is still fine.